orchard-core/src/main/scala/com/salesforce/mce/orchard/system/actor/ResourceMgr.scala

```diff
@@ -126,6 +134,11 @@ object ResourceMgr {
         ps.ctx.log.info(s"${ps.ctx.self} (idle) received GetResourceInstSpec($replyTo)")
         ps.database.sync(ps.resourceQuery.setRun())
         val instId = 1
+        ps.ctx.self ! CreateResourceInst(replyTo, instId)
+        Behaviors.same
```

Comment:
what if `idle` receives `CreateResourceInst` again?

Reply:
after it received the first `CreateResourceInst` it'll become running or finished state. It won't go back to idle anymore.